supplementary_style_guide/style_guidelines/grammar.adoc

```diff
@@ -83,5 +83,20 @@ As needed, adjust the conversational to _fairly conversational_ for an audience
 
 Red Hat Enterprise Linux 8 delivers a stable, secure, and consistent foundation across hybrid cloud deployments with the tools needed to deliver workloads faster with less effort.
 
+== Homographs
+A homograph is a word that is spelled the same as another word but has a different meaning when used as a different part of speech.
+Using homographs close together in a sentence or paragraph might confuse readers. 
+Therefore, when possible, avoid writing sentences that use homographs close to one another. 
+
+Homographs that might commonly appear in technical documentation include the following:
```

Comment:
Per ISG should have something after "following". Putting a suggested rewrite below, but feel free to adjust however you see fit.

```suggestion
The following list includes examples of homographs that might appear in technical documentation:
```

Reply:
fixed